.lsp/config.edn

```diff
@@ -0,0 +1 @@
+{:clean {:ns-inner-blocks-indentation :keep}}
```

Comment:
@yqrashawn, I haven't checked, but do you know if we can add a setting to the edn file excluding certain namespaces? Just so nobody sorts the `status-im2.core` namespace by accident in their text editors.